### ISSUE TITLE
Passing extension and lua filter at the same time?

### ISSUE THREAD

**Issue by owlonewolf** [pandoc/lua-filters#263]
I can't seem to combine the markdown extension Ext_mark (reads "==text==" as used in obsidian and highlights text in docx) and the lua filter include-files in the same pandoc conversion. I can make both work on their own, but when I combine both, the cmd flag `-f markdown+mark` is ignored.  The order of `-L` and `-f` doesn't change anything. 

@tarleb [suggested to ensure that the pandoc.read function uses markdown+mark as format](https://github.com/jgm/pandoc/issues/7743#issuecomment-1421040239). I've played around, but cannot figure out how to specify that in the .lua file

If I understood Tarleb correctly, I should specify the extension in line 91-95 of include-files.lua, but how?
```
local contents = pandoc.read(
          fh:read '*a',
          format,
          PANDOC_READER_OPTIONS
        ).blocks
```

**Comment by jgm** [comment]
It might matter what version of pandoc you're using, so let us know.

**Comment by tarleb** [comment]
Looking at the code, I think that adding `format="markdown+mark"` to the attributes of the include block should be enough.

**Comment by owlonewolf** [comment]
Thank you both for the help so far (and your great work). I'm using Pandoc 3.0. 

Tarleb, I tried that in line 93 of the lua, which gives me `Error .... .lua:93: ')' expected (to close '(' at line 91) near '='`. The lua filter works fine without it. 
Just in case I misunderstood you, I also tried in line 66 `local format = cb.attributes[format="markdown+mark"]`

Let me know if it seems I misunderstood the meaning of "attributes of the include block"
